mlflow/onnx.py

```diff
@@ -182,10 +199,49 @@ def _load_model(model_file):
 
 
 class _OnnxModelWrapper:
-    def __init__(self, path):
+    def __init__(self, path, providers=None):
         import onnxruntime
 
-        self.rt = onnxruntime.InferenceSession(path)
+        # Get the model meta data from the MLModel yaml file which may contain the providers specification
```

Comment:
```suggestion
        # Get the model meta data from the MLModel yaml file which may contain the 
        # providers specification
```
To pacify the lint requirement of 100 lines